pkg/proc/interface.go

```diff
@@ -95,31 +112,3 @@ type Checkpoint struct {
 	When  string
 	Where string
 }
-
-// Info is an interface that provides general information on the target.
-type Info interface {
```

Comment:
These interfaces were more for organization than to be directly used. I guess I'm ok with removing them, but how would you feel about simple unexporting them to make clear they aren't to be used outside this package?

Reply:
IMO there's so few methods that they make things harder to understand rather than clearer. But I don't feel stronger about it so if you want I'll revert.
Making them unexported would be worse because they wouldn't be shown in the godoc documentation.

Reply:
Ack, I don't feel strongly so SGTM.